tests/Dockerfile

```diff
@@ -46,28 +46,44 @@ RUN dnf -y update && \
     dnf -y builddep /tmp/cockpit.spec && \
     dnf clean all
 
-RUN mkdir -p /usr/local/bin /home/user /build
-ADD cockpit-verify vm-prep network-cockpit.xml /usr/local/bin/
+RUN mkdir -p /usr/local/bin /home/user /build /build/tmp /secrets /images
+ADD cockpit-tests install-service vm-prep network-cockpit.xml /usr/local/bin/
 
-RUN chown -R user /build /home/user
 RUN usermod -a -G mock user
 RUN echo 'user ALL=(ALL) NOPASSWD: ALL' >> /etc/sudoers
 RUN echo "config_opts['basedir'] = '/build/mock/'" >> /etc/mock/site-defaults.cfg
+RUN printf '[user]\n\t\nemail = cockpituous@cockpit-project.org\n\tname = Cockpituous\n' >/home/user/.gitconfig && \
+    ln -s /images /build/images && \
+    mkdir -p /home/user/.config /home/user/.ssh && \
+    ln -snf /secrets/ssh-config /home/user/.ssh/config && \
+    ln -snf /secrets/github-token /home/user/.config/github-token && \
+    git clone https://github.com/cockpit-project/cockpit /build/cockpit && \
+    cd /build/cockpit && npm install && \
+    chown -R user /build /secrets /images /home/user
 
 # Prevent us from screwing around with KVM settings in the container
 RUN touch /etc/modprobe.d/kvm-amd.conf && touch /etc/modprobe.d/kvm-intel.conf
 
-ENV TEST_IMAGES_RHEL http://file.rdu.redhat.com/~swalter/images/
+# Continue to work with a host defined cockpit1 network if necessary
+RUN chmod -v u+s /usr/libexec/qemu-bridge-helper && echo 'allow cockpit1' >> /etc/qemu/bridge.conf
+
 ENV XDG_CACHE_HOME /build/tmp
 ENV XDG_CONFIG_HOME /build/tmp
 ENV TMPDIR /build/tmp
 ENV TEMP /build/tmp
 ENV TEMPDIR /build/tmp
+ENV TEST_DATA /build
 
-VOLUME /home/user
-VOLUME /build
+VOLUME /secrets
+VOLUME /images
 
 USER user
 WORKDIR /build
-ENTRYPOINT ["/usr/local/bin/cockpit-verify"]
+ENTRYPOINT ["/usr/local/bin/cockpit-tests"]
 CMD ["--publish=fedorapeople.org", "--verbose"]
+
+# We execute the script in the host hence the use of chroot and appropriate piping of the script
```

Comment:
I'm afraid this comment doesn't really help - the command already makes that clear. I was wondering *why* this runs install-service through that wrapping -- does it make any difference compared to just running it directly on the host?